react-client/src/Components/Sidebars.jsx

```diff
@@ -31,7 +31,7 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
                 />
             </div>
             <div className="grid-container-Ratings-Spacing">
-            <   div  className="ratingsText-Side"> Design</div>
+            <   div  className="ratings-Text-Side"> Design</div>
```

Comment:
I think I commented on this Red arrow before. I don't think it's wrong, but git hub is highlighting it so I want to bring it to your attention.